data/powerup.xml

```diff
@@ -74,38 +74,123 @@
-       kart (first kart, top 33%, middle 33%, bottom 33% (but
-       not last), last kart. i
+       kart. There are defined value for first kart, last kart, and
+       at three equally spaced points between those.
+       For example, with 5 karts those points will match positon 2, 3, 4.
+       With 10 karts, they will correspond to 3,25 ; 5,5 ; 7,75.
+       It is not an issue for it to be a float, as the weights for positions
+       are linearly averaged.
        The order of items must correspond to powerup_manager.hpp.
-       The first line (w=...) corresponds to the weights of 
-       getting a single item, the second line (w-multi) to 
-       getting three identical items instead of just a single one.
-       E.g. as the first kart, you have 3/8 of a chance to get
-       a single bubble gum, 1/8 to get a bowling ball or switch,
-       and 3/8 to get a triple bubble gum.
-       'Global' items which affect all karts (switch) should
+
+       The first line (w1=...) corresponds to the weights of 
+       getting a specific item, the second line (w-multi1) to 
+       the weight at which it will yeld a triple item
+       rather than a single one
+
+       The probability to get an item is its weight divided by
+       sum of weights of all items (single AND multi). It is recommended 
+       to keep that sum equal to 200 to easily keep track of probabilities.
+
+       'Global' items which affect all karts (switch, parachute) should
        be quite rare, since otherwise the item might be used
        too often (compared with many items which will only
-       affect a karts or two).                                -->
+       affect a karts or two).
+
+       The distribution should give more similar item to different ranks
+       when there are few karts, but have more important differences when
+       there are more karts.                               -->
+  <!--      bubble  cake  bowl  zipper  plunger  switch swattr rubber para  anvil -->
+  <first  w1="28       0    60    20      45       15      32     0     0     0"
+    w-multi1=" 0       0     0     0       0        0       0     0     0     0"
+          w2="26      16    52    15      46       12      33     0     0     0"
+    w-multi2=" 0       0     0     0       0        0       0     0     0     0"
+          w3="24      12    58    10      54        8      34     0     0     0"
+    w-multi3=" 0       0     0     0       0        0       0     0     0     0"
+          w4="22       8    64     5      60        6      35     0     0     0"
+    w-multi4=" 0       0     0     0       0        0       0     0     0     0"
+          w5="20       0    74     0      66        4      36     0     0     0"
+    w-multi5=" 0       0     0     0       0        0       0     0     0     0" 
+          wf="35       0    25    35      25       15      25     0     0     0"
+    w-multif="20       0     0    20       0        0       0     0     0     0"  />
```

Comment:
What is "multif" ?

Reply:
It would be good to clarify that, at least with a comment, of by renaming it ("-ftl" maybe?)

Reply:
Comment added. I kept it as wf ; as the clarity difference is minimal with the comment and as it would mean having to add spaces to realign over 50 lines.